conductor-client/src/main/java/conductor/utils/ViewUtils.kt

```diff
@@ -0,0 +1,52 @@
+package conductor.utils
```

Comment:
nit: Should we add copyright here. I added a setting in IntelliJ to produce copyright to each new file in the conductor project. Can we share the setting in some form? Not blocker for this PR though.

Reply:
@amanjeetsingh150 sounds good, could you add this setting to the repo? Also, are there perhaps any Gradle plugins which do the same?